manifests/init.pp

```diff
@@ -106,15 +120,49 @@
   Boolean $candlepin_db_ssl = $::katello::params::candlepin_db_ssl,
   Boolean $candlepin_db_ssl_verify = $::katello::params::candlepin_db_ssl_verify,
   Boolean $candlepin_manage_db = $::katello::params::candlepin_manage_db,
+
+  Boolean $manage_candlepin = $::katello::params::manage_candlepin,
+  Boolean $manage_qpid = $::katello::params::manage_qpid,
+  Boolean $manage_pulp = $::katello::params::manage_pulp,
+  Boolean $manage_application = $::katello::params::manage_application,
+
+  String $candlepin_hostname = $::katello::params::candlepin_hostname,
+  String $qpid_hostname = $::katello::params::qpid_hostname,
+  String $pulp_hostname = $::katello::params::pulp_hostname,
 ) inherits katello::params {
-  include ::katello::repo
-  include ::katello::candlepin
-  include ::katello::qpid
-  include ::katello::pulp
-  Class['katello::repo'] -> Class['katello::pulp']
-  include ::katello::application
-  Class['katello::repo'] -> Class['katello::application']
-  Class['katello::candlepin'] -> Class['katello::application']
-
-  User<|title == apache|>{groups +> $user_groups}
+  $pulp_manage_httpd = ! $manage_application
+  $candlepin_url = "https://${candlepin_hostname}:8443/candlepin"
+  $qpid_url = "amqp:ssl:${qpid_hostname}:5671"
+  $pulp_url = "https://${pulp_hostname}/pulp/api/v2/"
+
+  if $manage_candlepin {
+    include ::katello::candlepin
+  }
+
+  if $manage_qpid {
+    include ::katello::qpid
+  }
+
+  if $manage_pulp {
+    unless $manage_candlepin {
+      fail('Pulp relies on candlepin for the CRL. Set $manage_candlepin to true.')
```

Comment:
Hrmm, we should look more into this because we don't enforce or use CRL.

Reply:
I was thinking of a switch to use local or a simple script to retrieve the CRL and reload apache if it changed but first I want to get the setup with 2 servers running.

Reply:
I'd really appreciate it, if this check would be removed. There might be users (me :-P) who want to ignore this dependency for starters.

Reply:
I hadn't arrived there yet and wanted to be safe. My thinking was a parameter like ```Enum['disabled', 'candlepin'] $pulp_crl``` that will allow us to add a script later that retrieves the CRL. Thoughts?

Reply:
Yep, this sounds like a good compromise. I like the idea.